cheroot/workers/monitor.py

```diff
@@ -0,0 +1,154 @@
+"""Monitor for threadpool."""
+__metaclass__ = type
+
+import threading
+import time
+
+from .dynpool import DynamicPoolResizer
+
+
+class BackgroundTask(threading.Thread):
+    """A subclass of threading.Thread whose run() method repeats.
+
+    Use this class for most repeating tasks. It uses time.sleep() to wait
+    for each interval, which isn't very responsive; that is, even if you call
+    self.cancel(), you'll have to wait until the sleep() call finishes before
+    the thread stops. To compensate, it defaults to being daemonic, which means
+    it won't delay stopping the whole process.
+    """
```

Comment:
It'd be useful to explain args with Sphinx syntax following this example: https://github.com/cherrypy/cheroot/blob/85a190f/cheroot/server.py#L431-L441

Reply:
Done